recruitments.html

```diff
@@ -115,89 +118,100 @@ <h2 class="orange text-center">
 				<p>
 					The challenges for AI recruitments have been hosted on HackerRank. The challenges
 					includes easy and hard challenges.
-					The challenge shall remain open till 10th September, 2019.
+					The challenge shall remain open till 30th November, 2020.
```

Comment:
I have to confirm with ritwik if the last date is 27th Nov, might have to change after that.